projects/schematics/src/shared/utils/file-utils.ts

```diff
@@ -26,7 +31,13 @@ export enum InsertDirection {
 
 export interface ClassType {
   className: string;
-  importPath: string;
+  importPath?: string;
+  staticType?: string;
```

Comment:
I don't think it is but I didn't want to call it "basicType" since you can pass objects. Also I thought "type" was too simple.

Reply:
It seems that this is an official name: https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#literal-inference

Maybe we should use, in order to avoid confusion.